### PR TITLE
Feature/id 304/delete account UI (warning)

### PR DESCRIPTION
**Pull request by DiegoFigueroaSevillano**
## AUTHORS: 
- @DiegoFigueroaSevillano 

 **WHAT?**

I Create the popup view that will be displayed when you want to delete our account, all this with its functionalities

**HOW?**

Make the layout of the view, together with this, implement your controller class in which you can give the buttons their respective functionality and then implement all this within the delete account button which is found within the user information view

 **WHY?** 

All this is done because it is important that the user is sure that he wants to delete his account and that the consequences of it are shown on the screen.

## CODE REVIEWERS/TESTERS

Denis Gandarillas
Emanuel Galindo

### CODE REVIEW

**Comment by denisgandel** [comment]
Screenshot from 2023-06-14 23-01-04

The "I'm sure!" button should be red, as this would denote that it is not a good option.

File: app/src/main/java/com/isc/hermes/AccountInformation.java [hunk 1]
```diff
@@ -5,11 +5,15 @@
...
 import android.view.View;
 
+import com.isc.hermes.controller.DeleteAccountWarningDialog;
+
 /**
  * This class represents the AccountInformation activity, which displays information about the account.
  */
 public class AccountInformation extends AppCompatActivity {
 
+    DeleteAccountWarningDialog dialog;
```
Comment (denisgandel) [inline app/src/main/java/com/isc/hermes/AccountInformation.java:15]: Being an attribute of the class, you should give it a private access.

File: app/src/main/java/com/isc/hermes/controller/DeleteAccountWarningDialog.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+/**
+ * This is the controller class for the notification popup when we try to delete our account
+ * It extends from the Dialog class
+ * It implements a listener which will listen to the option we choose.
+ */
+public class DeleteAccountWarningDialog extends Dialog implements View.OnClickListener {
+    private Button imSureButton;
+    private Button betterNotButton;
+    private Activity activity;
+    GoogleAuthentication googleAuthentication;
```
Comment (denisgandel) [inline app/src/main/java/com/isc/hermes/controller/DeleteAccountWarningDialog.java:23]: Being an attribute of the class, you should give it a private access.

**Comment by DiegoFigueroaSevillano** [comment]
@denisgandel In this case it would not make much sense for it to be red because it is not a "correct option" since if the user clicked on delete account it is the option they are looking for

**Comment by denisgandel** [comment]
image

Verify that you have not removed or changed any functionality, because when I enter the Account section, it does not send me to the Account activity to test your implementation.

**Comment by denisgandel** [comment]
@DiegoFigueroaSevillano Yes, but it would not be an option sought by us, because we would end up losing a user.

**Comment by DiegoFigueroaSevillano** [comment]
@denisgandel I did not touch any of the features that take us to the user information tab, to do my part I just ran it from user information view. So that functionality is either not implemented or is implemented in another branch

**Comment by denisgandel** [comment]
@DiegoFigueroaSevillano Very well that part must be from another task or US.

**Review by denisgandel — APPROVED** [review]
I don't have more comments

**Pull request merged**